### DISCUSSION TITLE
Welcome to JingOS Discussions!

### DISCUSSION THREAD

**Discussion by JingOS-team** [JingOS-team/JingOS#78] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dioaffriza** [comment]
Hello, I am Affriza from Indonesia. Just a normal desktop programmer that mainly using Qt framework. 

There's several questions I want to ask:

- Is this project is still alive?
- What's your connection with CutefishOS?
- Is the X86 port officially dead?
